src/temp/stories/ProjectFilter.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import Filter from '../Project/Filter';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof Filter> = {
+  component: Filter,
+};
+
+export default meta;
+type Story = StoryObj<typeof Filter>;
+
+export const inActive: Story = {
+  args: {
```

Comment:
inActive is one word. So this should become `Inactive` instead of `inActive` or `InActive`